tests/Dockerfile

```diff
@@ -2,9 +2,9 @@ FROM mcr.microsoft.com/playwright:v1.40.0-jammy
 
 ENV HEADLESS=true
 
-RUN apt-get update && apt-get install --yes python3-pip
-RUN pip install pytest playwright pytest-playwright faker
-RUN playwright install
+RUN apt-get update && apt-get install --yes python3-poetry
+RUN poetry install
+RUN poetry run playwright install
```

Comment:
At the first moment I was missing the installation of playwright's dependencies, e.g. browser(s).
Then I realized again, you're using another source image than me coming along with everything you need to play with playwright. I like it.